kpi/serializers/v2/asset_permission_assignment.py

```diff
@@ -65,9 +64,7 @@ def create(self, validated_data):
         user = validated_data['user']
         asset = validated_data['asset']
         if asset.owner_id == user.id:
-            raise serializers.ValidationError(
-                {'user': t(ASSIGN_OWNER_ERROR_MESSAGE)}
-            )
+            raise serializers.ValidationError({'user': t(ASSIGN_OWNER_ERROR_MESSAGE)})
```

Comment:
It's going over the 80 characters limit (but below 90*). Did you do this in purpose or your linter is configured this way? 

`*` Old Kobo Python coding convention discussion: We usually break at 80 but allow up to 90 when it's only for few characters.

Reply:
It appears screwed up my local config to use default 88 when it was already set to 80. Using the formatter with 80 is OK right? I will update. Note the formatter greatly helps in the (soon to be) large permission dict (") and json (') that we need in docs and tests.